.github/CONTRIBUTING.md

```diff
@@ -59,6 +59,7 @@ system with CMake 3.14+:
 ```bash
 python3 -m venv venv
 source venv/bin/activate
+pip install -U pip wheel
```

Comment:
```suggestion
python3 -m venv --upgrade-deps venv
source venv/bin/activate
```

Not sure when this was added, but guessing it's been around for a while? Using `virutalenv` ensures a recent versions of everything and is faster, but is not builtin.

Reply:
Ouch, this was added in Python 3.9. One of the problems of being on a Mac... ;)

```console
$ docker run --rm -it quay.io/pypa/manylinux_2_24_x86_64:latest bash
# python3.6 -m venv --help
# python3.7 -m venv --help
# python3.8 -m venv --help
# python3.9 -m venv --help
```